setup.py

```diff
@@ -30,7 +30,7 @@ def plotly_js_version():
 
 
 def readme():
-    with open('README.rst') as f:
+    with open('README.md') as f:
```

Comment:
This is what's on Github by default and is much more up to date than the one on PyPI